src/watchdog/watchmedo.py

```diff
@@ -154,6 +154,24 @@ def schedule_tricks(observer, tricks, pathname, recursive):
 @arg('--recursive',
      default=True,
      help='recursively monitor paths')
+@arg('--debug-force-polling',
+     default=False,
+     help='[debug] forces polling')
+@arg('--debug-force-kqueue',
+     default=False,
+     help='[debug] forces BSD kqueue(2)')
+@arg('--debug-force-winapi',
+     default=False,
+     help='[debug] forces Windows API')
+@arg('--debug-force-winapi-async',
+     default=False,
+     help='[debug] forces Windows API + I/O completion')
+@arg('--debug-force-fsevents',
```

Comment:
Do you mind also adding `--debug-force-fsevents2`?

Reply:
I can but none of the other `watchmedo` commands currently has that, and I don't want this PR to be denied on the grounds that I'm adding in totally new functionality without review.

Reply:
You are right, let's do without it.

Reply:
I think it might be worthwhile reviewing the usage of the `--debug-force-*` flags and the automatic selection of observers; as was mentioned in #779 it might be better (at least for tricks) to be able to specify the observer on a per-trick basis, which would remove the requirements to duplicate these flags across each CLI command.

Reply:
Indeed, and if you want to have a look, I would love to review such a PR :)
Maybe could we finish this one, and merge it, and then improve the situation later.